### DISCUSSION TITLE
New labels for different languages

### DISCUSSION THREAD

**Discussion by Arsenic-ATG** [MakeContributions/DSA#51] [Ideas]
now all the new programs are labeled as `enhancement`

because of the obvious nature of this repository, we would be having different contributions for same/similar algorithms and data structures so wouldn't it be better to have different labels for different languages like one for C++, one for python etc. so that it becomes easier to review and manage PR's and issues about implementation of different algorithms.

for example : -
let's say I decide go to `pull requests` section and decide to review all contributions done in C++ language then instead of going over each and every pull request to see which language was used there, I can just filter the PRs to just show me the ones with `C++` label. 

status update : - 
i have added labels for the languages in which contributions are done till now and labeled all the PRs accordingly.

**Comment by Arsenic-ATG** [comment]
I could have done it but thought it would be a better idea to consult members to see what they have to say about it.

**Comment by ghost** [comment]
I agree with this. Will you add the labels and classify old PRs, @Arsenic-ATG?

**Comment by ghost** [comment]
We will keep both `enhancement` and the specific language label (eg. `python`).
